js/indicators/price-channel.src.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+import H from '../parts/Globals.js';
+import '../parts/Utilities.js';
+import reduceArrayMixin from '../mixins/reduce-array.js';
+import requiredIndicatorMixin from '../mixins/indicator-required.js';
+
+var minInArray = reduceArrayMixin.minInArray,
+    maxInArray = reduceArrayMixin.maxInArray,
+    BB = H.seriesTypes.bb,
+    parentLoaded = requiredIndicatorMixin.isParentIndicatorLoaded;
+
+H.seriesType('pc', 'bb',
+    /**
+     * Price channel (PC). This series requires the `linkedTo` option to be
+     * set and should be loaded after the `stock/indicators/indicators.js`
+     * and `stock/indicators/bollinger-bands.js` files.
+     *
+     * @extends plotOptions.bb
+     * @product highstock
+     * @sample {highstock} stock/indicators/price-channel Price Channel
+     * @excluding
+     *        allAreas, colorAxis, compare, compareBase, joinBy, keys, stacking,
+     *        showInNavigator, navigatorOptions, pointInterval,
+     *        pointIntervalUnit, pointPlacement, pointRange, pointStart, joinBy
+     * @since 7.0.0
+     * @optionparent plotOptions.pc
+     */
+    {
+        name: 'Price Channel (20)',
```

Comment:
Name should be build from `nameBase` and `nameComponents`, see for example `ZigZag` props.

Reply:
Fixed.